common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
```

Comment:
@e-kolpakov ^ Checkout this discussion and this PR - may provide a simpler approach for https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/86 though perhaps it's too late.